certify-default.properties

```diff
@@ -144,3 +144,4 @@ mosip.certify.cache.expire-in-seconds={'userinfo': ${mosip.certify.access-token-
 
 mosip.certify.domain.url=
 mosip.certify.key-values={}
+mosip.certify.database.name=
```

Comment:
we can give a default value like mosip_certify for this

Reply:
added this